firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -269,6 +269,11 @@ class ToolbarMiddleware: FeatureFlaggable {
                                               windowUUID: action.windowUUID,
                                               actionType: GeneralBrowserActionType.showReloadLongPressAction)
             store.dispatch(action)
+        case .newTab:
+            let action = GeneralBrowserAction(windowUUID: action.windowUUID,
```

Comment:
note to self: if we're only dispatching actions, should that live in the middleware 🤔 ? Something, I need to ponder, but also welcome ideas / thoughts.

Reply:
added to redux guidelines, resolving here.